src/syft/lib/sympc/session_util.py

```diff
@@ -25,12 +25,16 @@ def protobuf_session_serializer(session: Session) -> MPCSession_PB:
         (length_nr_parties + 7) // 8, byteorder="big"
     )
 
+    protocol = session.protocol.__name__
+    protocol_serialized = str.encode(protocol)
```

Comment:
Q: @tudorcebere do we need to encode the string or we can leave it simple like `protocol`?

Reply:
As long as it's a str, I think you can leave it be

Reply:
Then do not encode it